test.js

```diff
@@ -2,8 +2,10 @@
  * Module dependencies.
  */
 
-var assert = require('assert');
-var validate = require('./');
+var assert = require('assert')
+var validate = require('./')
+
+assert.deepStrictEqual = require('is-equal-shallow')
```

Comment:
> attaching properties to core libs is generally not a good idea

why not? :D actually its absolutely the same and iojs have it, so its like polyfill.